src/main/java/frc/robot/commands/DrivetrainPIDTurnDelta.java

```diff
@@ -64,6 +75,6 @@ public void execute() {
         turnThrottle = (targetAngle - currentAngle) * pGain.get() - (currentAngularRate) * dGain.get();
 
         //Run motors according to the output of PD
-        drivebase.tankDrive(-turnThrottle, turnThrottle, false);
+        drivebase.tankDrive(-turnThrottle + forwardSpeed, turnThrottle + forwardSpeed, false);
```

Comment:
This might be an odd case, but since I think tankDrive clamps it's input to `0 <= x <= 1` you'll run in to some issues when `turnThrottle + forwardSpeed > 1` since the clamp will chop off part of your speed difference.  
I imagine it will only affect how fast you turn though.  

Also I know `A + B` is the same as `B + A` but I think it would look better with the variables swapped around 😛

Reply:
I think that it would be a rare case, since it would only matter a lot if we are going at max velocity, which I feel like we wouldn't do. I also feel like when used with vision they wouldn't want the robot moving at max speed since that would cause motion blur.